scratch3/blocks.js

```diff
@@ -703,7 +703,7 @@ ScriptView.prototype.draw = function(inside) {
   }
   this.height = y + 1
   if (!inside && !this.isFinal) {
-    this.height += block.hasPuzzle ? 44 : 36
+    this.height += block.hasPuzzle ? 8 : 0
```

Comment:
Why not 9 and 1? Isn't this cutting the bottom part by one?

Reply:
8 and 0 seem not to cut the bottom part (with my own eye), but, I feel we don’t need to reduce spaces aggressively.
Out of curiosity, how did you select 9 and 1?

Reply:
My mistake. This is correct. Should be ready to merge